src/Component/Result/Facet/JsonRange.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace Solarium\Component\Result\Facet;
+
+/**
+ * JsonRange field facet result.
+ *
+ * The range values are a dataset of multiple rows, in each row a
+ * value and its count. You can access the values as an array using
+ * {@link getValues()} or iterate this object.
+ * The additional properties of before, after, and between are only avilable if the initial request has the 'other' param set.
+ * See https://lucene.apache.org/solr/guide/json-facet-api.html#range-facet-parameters
+ */
+class JsonRange extends Buckets
+{
+    /**
+     * Count of all records with field values lower then lower bound of the first range.
+     *
+     * @var int|null
+     */
+    protected $before;
+
+    /**
+     * Count of all records with field values greater then the upper bound of the last range.
+     *
+     * @var int|null
+     */
+    protected $after;
+
+    /**
+     * Count all records with field values between the start and end bounds of all ranges.
+     *
+     * @var int|null
+     */
+    protected $between;
+
+    /**
+     * Value array.
+     *
+     * @var Bucket[]
+     */
+    protected $buckets;
+
+    /**
+     * numBuckets.
+     *
+     * @var int|null
+     */
+    protected $numBuckets;
+
+    /**
+     * Constructor.
+     *
+     * @param Bucket[] $buckets
+     * @param int|null $numBuckets
```

Comment:
You've probably overlooked this in a copy/paste. 😉

```suggestion
     * @param Bucket[] $buckets
     * @param int|null $before
     * @param int|null $after
     * @param int|null $between
```

Reply:
Thanks, yep I used Buckets as a base and just realised I left in the other parent methods too! Will fix now…